exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "exercise": "revere-string",
+  "version": "1.0.1",
+  "comments": [
+    "The string argument cases possible matches are passed in as",
+    "individual arguments rather than arrays. Languages can include",
+    "these string argument cases if passing individual arguments is",
+    "idiomatic in that language."
+  ],
+  "cases": [
+    {
+      "description": "no matches",
+      "property": "reverse-string",
+      "subject": "car",
+      "candidates": ["girl", "Japan", "bear", "candy"],
+      "expected": []
+    },
+    {
+      "description": "detects simple reverse string",
+      "property": "reverse-string",
+      "subject": "ant",
+      "candidates": ["tna"],
+      "expected": ["tan"]
+    },
```

Comment:
As well, what is candidates for, and why do you expect an array of strings to be returned?